leaflet-treering.js

```diff
@@ -1651,7 +1649,8 @@ function InsertPoint(Lt) {
         Lt.visualAsset.reload();
       }
 
-      this.disable();
+      /* Disables after one click
+      this.disable(); */
```

Comment:
generally if we're unlikely to re-activate this code, I don't like to leave dead code sitting around. This won't be clear to future self.
If you think we might want to reenable this in the future, I'd write it something like:
// uncommenting the line below will cause the "add point" tool to disable itself after a point is added.
// otherwise it'll stay active until the user manually disables the tool
// this.disable();